src/client/lazy-app/Compress/Modal/style.css

```diff
@@ -0,0 +1,174 @@
+.modal-overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  backdrop-filter: blur(5px);
+  -webkit-backdrop-filter: blur(5px);
+  z-index: 1;
+
+  user-select: none;
+  pointer-events: none;
+  opacity: 0;
+
+  transition: opacity 250ms ease;
+}
+
+.modal-shown {
+  user-select: auto;
+  pointer-events: auto;
+
+  opacity: 1;
+}
+
+.modal {
+  width: 70vw;
+  max-width: 60ch;
+  max-height: 70vh;
+
+  border: 4px solid black;
+
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+
+  font-size: 1.5em;
+
+  border-radius: 10px;
+  overflow: hidden;
+
+  background-color: hsl(250deg, 100%, 90%);
+}
+
+@media (max-width: 720px) {
+  .modal {
+    width: 90%;
+    max-width: none;
+    font-size: 1.25em;
+  }
+}
+
+@media (max-width: 480px) {
+  .modal {
+    font-size: 1em;
+  }
+}
+
+.header,
+.footer {
+  max-height: 100%;
+  padding: 1em;
+
+  background-color: rgba(0, 0, 0, 6%);
+}
+
+.header {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  align-items: center;
+  gap: 0.5em;
+  border-bottom: 1px solid rgba(0, 0, 0, 8%);
+
+  & > * {
+    height: 100%;
+    font-size: 1.5em;
+  }
```

Comment:
This seems a bit dangerous. I think instead of setting a height on each child, setting `align-content: stretch;` on the header should do the trick. Same with the font size, which is automatically inherited.